core/src/dird/ua_status.cc

```diff
@@ -513,46 +532,53 @@ static bool DoSubscriptionStatus(UaContext* ua)
     return false;
   }
 
+  char now[30] = {0};
+  bstrftime(now, sizeof(now), (utime_t)time(NULL), "%F %T");
+
   if (kw_all || kw_detail) {
+    ua->send->ObjectKeyValue(
+        "version", "Bareos version: ", kBareosVersionStrings.Full, "%s");
+    ua->send->ObjectKeyValue("os", kBareosVersionStrings.GetOsInfo(),
+                             " (%s)\n");
+    ua->send->ObjectKeyValue("binary-info",
+                             "Binary info: ", kBareosVersionStrings.BinaryInfo,
+                             "%s\n");
+    ua->send->ObjectKeyValue("date", "Date: ", now, "%s\n");
```

Comment:
maybe we want that part really only when "all" was set?

Reply:
Why? As "all" can produce confusing outputs (unknown section), currently we switched WebUI to show "status subscription detail" instead of "status subscription all" and I'd like to see this information in the WebUI report.